test/replication-luatest/quorum_master_test.lua

```diff
@@ -41,14 +40,10 @@ pg.before_test('test_master_master_works', function(cg)
         box.schema.space.create('test', {engine = params.engine})
         box.space.test:create_index('primary')
     end
-    cg.cluster:exec_on_leader(bootstrap_function, {cg.params})
+    cg.cluster:get_leader():exec(bootstrap_function, {cg.params})
 
 end)
 
-pg.after_test('test_master_master_works', function(cg)
-    cg.cluster:drop({cg.master_quorum1, cg.master_quorum2})
```

Comment:
Why do you remove this part completely? In other tests it stays as `cg.cluster:drop()`.

Reply:
Because we have 

```lua
pg.after_each(function(cg)
    cg.cluster:drop()
end)
```

at L31, moreover, this is wrong usage of `drop()`.